autoPyTorch/api/base_task.py

```diff
@@ -610,9 +619,9 @@ def _close_dask_client(self) -> None:
             None
         """
         if (
-            hasattr(self, '_is_dask_client_internally_created')
-            and self._is_dask_client_internally_created
-            and self._dask_client
+                hasattr(self, '_is_dask_client_internally_created')
+                and self._is_dask_client_internally_created
```

Comment:
Please refer to [this](https://github.com/automl/Auto-PyTorch/pull/434/files#r895915491)